### PR TITLE
Add dockerfile

### LINKED ISSUE #9: Add support for different screen resolutions

### LINKED ISSUE #10: Create docker image

### PR DESCRIPTION
**Pull request by tlazor**
This PR adds a dockerfile for easier linux development and support for different screen resolutions

Closes #9 and closes #10

### CODE REVIEW

File: apex_ocr/__main__.py [hunk 1]
```diff
@@ -33,11 +36,14 @@ def main(filepath: str):
                 ]
             )
 
+        #  Assumes all files are the same size
+        scale_rois(Image.open(file_list[0]).size)
```
Comment (tlazor) [inline apex_ocr/__main__.py:40]: Future todo, handle directories of screenshots with different resolutions

File: compose.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    network_mode: "host"
+    environment:
+    - DISPLAY
+    volumes:
+      - ./data:/home/apex/apex-ocr/data
+      - /tmp/.X11-unix:/tmp/.X11-unix:ro
+      - /run/user/1001/pulse:/run/user/1001/pulse:ro
+      - $HOME/.config/pulse:/usr/src/app/.config/pulse
+    devices:
+      - /dev/snd
```
Comment (tlazor) [inline compose.yml:14]: Currently there is still an issue getting the beep to work. I will leave that for future work

File: cache_paddle.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from paddleocr import PaddleOCR
+
+try:
+    PaddleOCR(
+                use_angle_cls=True, lang="en", show_log=False, debug=False
+            )
+except ValueError:
+    pass
```
Comment (tlazor) [inline cache_paddle.py:8]: This is an ugly way to cache the model files that are downloaded at runtime, probably we can just copy those files to their correct location. Leaving for future work

**Comment by tlazor** [comment]
There are a number of squad summary screenshots that are misidentified as unknown summary page. Looking into that still

File: requirements.txt [hunk 4]
```diff
@@ -103,7 +103,8 @@ rarfile==4.0
 requests==2.28.2
 rich==13.3.4
 scikit-image==0.20.0
-scipy==1.10.1
+scipy==1.9.1
+screeninfo==0.8.1
 semantic-version==2.10.0
 shapely==2.0.1
 six==1.16.0
```
Comment (eqnguyen) [inline requirements.txt:110]: I think joblib is missing from this requirements.txt.

File: apex_ocr/config.py [hunk 5]
```diff
@@ -4,16 +4,28 @@
...
+
+PERSONAL_STATS_FILE =  DATA_DIRECTORY / "personal_stats.csv"
 SQUAD_STATS_FILE = DATA_DIRECTORY / "squad_stats.csv"
 
-# Bounding boxes for image grabs
-TOP_SCREEN = (0, 0, 1920, 250)
-PERSONAL_SQUAD_PLACED = (1450, 0, 1720, 200)
-XP_BREAKDOWN = (200, 200, 1024, 625)
+for m in get_monitors():
+    primary_monitor = None
```
Comment (eqnguyen) [inline apex_ocr/config.py:17]: y is this here

**Pull request merged**